docs/INSTALLATION.md

```diff
@@ -1,10 +1,10 @@
 ## Dart
 * [Install Dart](https://www.dartlang.org/install)
 * Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* From a new command window or terminal, type `dart --version` to verify that Dart is now available.
 
 ## Dart Plugin for your IDE or Editor
-* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools).
+* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools) if it exists for your editor of choice.
 
 ## Setting up Dart on your IDE
```

Comment:
Is this section necessary?

We basically want to make sure they have dart installed and is present in PATH env variable. Configuring IDE or Editors should be either optional tips/hints section or completely omitted. Since anyone contributing must have some experience with dart  Even if its a student(completely new to dart) and wants to report a bug or help us improve something the previous section "Dart Plugin for..."  should suffice.

Reply:
Everything under docs will be shown on as the sections for exercism.io/languages/dart .

So these docs are meant for folks who could be new to Dart, but want to try out some exercises to experienced folks.

Some IDEs may not catch that you are using Dart and thus won't ask for the path to the Dart SDK. So I thought it was a safe addition.

Who knows, they could be new to programming in general and thus don't know what an IDE is. So maybe we could flesh that out?